parm/use_cases/model_applications/pbl/PointStat_fcstHRRR_obsAMDAR_PBLH_PyEmbed.conf

```diff
@@ -0,0 +1,194 @@
+[config]
+
+# Documentation for this use case can be found at
+# https://metplus.readthedocs.io/en/latest/generated/met_tool_wrapper/PointStat/PointStat_python_embedding_obs.html
+
+# For additional information, please see the METplus Users Guide.
+# https://metplus.readthedocs.io/en/latest/Users_Guide
+
+###
+# Processes to run
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#process-list
+###
+
+PROCESS_LIST = GenVxMask, PointStat
+
+###
+# Time Info
+# LOOP_BY options are INIT, VALID, RETRO, and REALTIME
+# If set to INIT or RETRO:
+#   INIT_TIME_FMT, INIT_BEG, INIT_END, and INIT_INCREMENT must also be set
+# If set to VALID or REALTIME:
+#   VALID_TIME_FMT, VALID_BEG, VALID_END, and VALID_INCREMENT must also be set
+# LEAD_SEQ is the list of forecast leads to process
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#timing-control
+###
+
+LOOP_BY = INIT
+INIT_TIME_FMT = %Y%m%d%H
+INIT_BEG = 2022070108
+INIT_END = 2022070108
+#INIT_BEG = 2022070701
+#INIT_END = 2022073123
+INIT_INCREMENT = 1H
+
+LEAD_SEQ = 12
+#LEAD_SEQ = 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18
+
+# USER SETTINGS HERE
+# CUSTOM_LOOP_LIST = DENVER, DALLAS, BOSTON, MINNEAPOLIS   [list of airports sepearated by commas; each needs a mask file]
+# PY_SOUNDING_FLAG = ALL, ASC, DESC  [only one value here]
+# PY_PT_DELTA =  Potential Temperature delta setting (K) [usually 1.0-2.5]
+# Valid_Time(YYYYMMDD_HHMMSS)
+CUSTOM_LOOP_LIST = DENVER
+PY_SOUNDING_FLAG = ALL  
+PY_PT_DELTA = 1.25
+PY_VAL_TIME = {valid?fmt=%Y%m%d_%H0000}
+
+###
+# File I/O
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#directory-and-filename-template-info
+###
+
+GEN_VX_MASK_INPUT_DIR = {INPUT_BASE}/model_applications/pbl
+GEN_VX_MASK_INPUT_TEMPLATE = pblh_22{init?fmt=%j}{init?fmt=%H}0000{lead?fmt=%HH}
+GEN_VX_MASK_INPUT_MASK_DIR = {INPUT_BASE}/model_applications/pbl
+GEN_VX_MASK_INPUT_MASK_TEMPLATE = met_mask_{custom?fmt=%s}.txt
+
+GEN_VX_MASK_OUTPUT_DIR = {INPUT_BASE}/model_applications/pbl
+GEN_VX_MASK_OUTPUT_TEMPLATE = {INPUT_BASE}/model_applications/pbl/met_mask_{custom?fmt=%s}_100km.nc
+
+FCST_POINT_STAT_INPUT_DIR = {INPUT_BASE}/model_applications/pbl
+FCST_POINT_STAT_INPUT_TEMPLATE = {init?fmt=%Y%m%d%H}/pblh_22{init?fmt=%j}{init?fmt=%H}0000{lead?fmt=%HH}
+
+OBS_POINT_STAT_INPUT_DIR =
+OBS_POINT_STAT_INPUT_TEMPLATE = PYTHON_NUMPY= {INPUT_BASE}/model_applications/pbl/PointStat_fcstHRRR_obsAMDAR_PBLH_PyEmbed/calc_amdar_pblh.py {INPUT_BASE}/model_applications/pbl/22{valid?fmt=%j}{valid?fmt=%H}00q.cdf
```

Comment:
The push failed because the remote contains work I don't have locally. Do I 'git pull" first ?

Reply:
sent you a meeting invitation

Reply:
looks like you weren't there. we may be okay now. I did the following:

1) git commit -m "changed OBS path in conf file"
[feature_2198_useCase_AMDAR_pbl 1798cdb2] changed OBS path in conf file
 1 file changed, 1 insertion(+), 1 deletion(-)
jetfe1:/lfs1/BMC/wrfruc/jenglish/METplus/parm/use_cases/model_applications/pbl>git push origin feature_2198_useCase_AMDAR_pbl
Username for 'https://github.com': jason-english
Password for 'https://jason-english@github.com': 
To https://github.com/dtcenter/METplus
 ! [rejected]          feature_2198_useCase_AMDAR_pbl -> feature_2198_useCase_AMDAR_pbl (fetch first)
error: failed to push some refs to 'https://github.com/dtcenter/METplus'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

2) git fetch
remote: Enumerating objects: 153, done.
remote: Counting objects: 100% (153/153), done.
remote: Compressing objects: 100% (94/94), done.
remote: Total 153 (delta 74), reused 97 (delta 56), pack-reused 0
Receiving objects: 100% (153/153), 155.98 KiB | 3.63 MiB/s, done.
Resolving deltas: 100% (74/74), completed with 39 local objects.
From https://github.com/dtcenter/METplus
   98d355c2..ada08e94  feature_2198_useCase_AMDAR_pbl -> origin/feature_2198_useCase_AMDAR_pbl
   9f05ad44..0ca7584c  develop                        -> origin/develop
   d77358d4..bff2197e  develop-ref                    -> origin/develop-ref
 * [new branch]        feature_1667_updating_overview -> origin/feature_1667_updating_overview
   585a9038..4cbfed81  feature_2136_cross_spectra     -> origin/feature_2136_cross_spectra
   243d0782..f619ce9d  feature_2188_useCase_PANDA-C   -> origin/feature_2188_useCase_PANDA-C
 * [new tag]           v5.1.0-rc1                     -> v5.1.0-rc1

3) git status
On branch feature_2198_useCase_AMDAR_pbl
Your branch and 'origin/feature_2198_useCase_AMDAR_pbl' have diverged,
and have 1 and 1 different commits each, respectively.
  (use "git pull" to merge the remote branch into yours)

4) git pull
Merge made by the 'recursive' strategy.
 .github/parm/use_case_groups.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
jetfe1:/lfs1/BMC/wrfruc/jenglish/METplus>git status
On branch feature_2198_useCase_AMDAR_pbl
Your branch is ahead of 'origin/feature_2198_useCase_AMDAR_pbl' by 2 commits.
  (use "git push" to publish your local commits)
nothing to commit, working tree clean

5) git push
Username for 'https://github.com': jason-english
Password for 'https://jason-english@github.com': 
Enumerating objects: 21, done.
Counting objects: 100% (16/16), done.
Delta compression using up to 80 threads.
Compressing objects: 100% (9/9), done.
Writing objects: 100% (9/9), 910 bytes | 455.00 KiB/s, done.
Total 9 (delta 6), reused 0 (delta 0)
remote: Resolving deltas: 100% (6/6), completed with 6 local objects.
To https://github.com/dtcenter/METplus
   ada08e94..fee0b4d3  feature_2198_useCase_AMDAR_pbl -> feature_2198_useCase_AMDAR_pbl

Reply:
I've also fixed the FCST and mask subdirectories in the conf file, and committed that just now.